find_suggested_phrases.py

```diff
@@ -86,50 +88,52 @@ def match_abbrevs_to_phrases(results: List[tuple], presets : Dict[str, str]) ->
     return shortcut_dict
 
 
-def load_corpus(corpus_path="data/corpus/") -> List[str]:
+def load_corpus(path: str, extensions: set[str]) -> Iterable[str]:
     """Load all txt files under data/corpus, and return as a list of strings"""
     all_lines = []
     found_data = False
-    for filename in os.listdir(corpus_path):
-        if filename.endswith(".txt"):
-            found_data = True
-            with open(os.path.join(corpus_path, filename), 'r', encoding="utf8") as f:
-                all_lines.extend(f.readlines())
-    if not found_data:
-        print("Warning: No txt files found in data/corpus/")
-    all_lines = [line.strip() for line in all_lines]
+    for dirpath, _dirnames, filenames in os.walk(path):
+        for filename in filenames:
+            if Path(filename).suffix in extensions:
+                found_data = True
+                lines = (Path(dirpath) / filename).read_text().splitlines(keepends=False)
+                yield from lines
+    if not all_lines:
```

Comment:
is `all_lines` used?

Reply:
Sorry forgot to complete this, fixed now